setup.cfg

```diff
@@ -46,6 +47,7 @@ install_requires =
     pytz>=2021.3
     requests>=2.26.0
     rp2>=1.7.1
+    urllib3<2 # 2+ is incompatible with Python ~3.9
```

Comment:
Why don't we update the supported Pythons to 3.10...3.13?

Reply:
In this PR?
I was going to do separately, but I guess it doesn't matter.

Reply:
This one should be good too.